client/src/api/jobApi.js

```diff
@@ -47,7 +47,7 @@ export const fetchJobByIdAPI = async (authToken, id) => {
 
 export const addJobAPI = async (authToken, newJobData) => {
   try {
-    await axios.post(`${API_URL}/postjob`, newJobData, {
+    await axios.post(`${API_URL}/jobs`, newJobData, {
```

Comment:
Check if /jobs exists and if it is the right API.

Reply:
This is the wrong API call.

Reply:
Changed Api /jobs to /postjob path.